utils/test_harness/tools/src/test_harness_metric.cpp

```diff
@@ -1022,22 +1022,24 @@ void metric_validate_streamer_marker_data(
             (metricTypedValue.value.ui64 != 0)) {
           LOG_INFO << "Valid Streamer Marker found with value: "
                    << metricTypedValue.value.ui64;
-          EXPECT_EQ(metricTypedValue.value.ui64,
-                    streamerMarkerValues[streamerMarkerValuesIndex]);
+          EXPECT_EQ(streamerMarkerValues[streamerMarkerValuesIndex],
```

Comment:
Why is this changed ?

Reply:
It's recommended to have actual value as first argument and expected value as second, Thank you.